hack/verify-govet.sh

```diff
@@ -18,4 +18,4 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
-go vet $(go list ./... | grep -v vendor)
+go vet $(go list ./... | grep -v 'k8s.io/test-infra/testgrid')
```

Comment:
can we drop a TODO or followup issue for testgrid?

Reply:
I guess. this one isn't really doable in any trivial fashion because the code doesn't even build with go @fejta

Reply:
govet breaks on importing a package that doesn't exist (because it's bazel generated)

Reply:
filed https://github.com/kubernetes/test-infra/issues/9749